### ISSUE TITLE
typo in chat test

### ISSUE THREAD

**Issue by onemen** [epicweb-dev/rocket-rental#1]
https://github.com/epicweb-dev/rocket-rental/blob/82e10fbe32e13575a999c21a30949f32c7052a48/tests/chat.test.ts#L18

line 13 and 18 are the same

```diff
-   await expect(page1).toHaveURL(/\/chats\/.+/)
+   await expect(page2).toHaveURL(/\/chats\/.+/) 
```

**Comment by onemen** [comment]
multi-user chat test pass when all test runs together with `npm run test:e2e:dev` the result is:
2 failed
    [chromium] › onboarding.test.ts:18:1 › onboarding 
    [chromium] › onboarding.test.ts:110:1 › reset password

2 passed
    login as existing user
    multi-user chat
    
when i run only the chat test with `npm run test:e2e:dev -- chat` the result is:
```
Error: expect(received).toBeVisible()
Call log:
  - expect.toBeVisible with timeout 5000ms
  - waiting for selector "role=listitem >> text=laudantium in"

    at C:\Users\Hadar\projects\rocket-rental\tests\chat.test.ts:45:79
```

**Comment by onemen** [comment]
fixed by c62c62c6ede9e4b28bcf5d3e3a62a033ceaf5cd5
